src/utils/checks/oneTxMustBeUpgraded.ts

```diff
@@ -0,0 +1,13 @@
+import { Extension, ONE_TX_PAYMENT_VERSION_LATEST } from '@colony/colony-js';
```

Comment:
On a different note, would it make sense to put this in the existing `utils/extensions` file or to create a separate subfolder for extensions utils rather than scatter them around?

Reply:
Sure, since `utils/extension` is small right now, I moved `oneTxMustBeUpgraded` into that file. I think we can create a subfolder later if necessary.